packages/core/src/app/address/AddressSelect.tsx

```diff
@@ -38,7 +39,7 @@ const AddressSelectMenu: FunctionComponent<AddressSelectProps> = ({
         {addresses.map((address) => (
             <li className="dropdown-menu-item dropdown-menu-item--select" key={address.id}>
                 <a href="#" onClick={preventDefault(() => onSelectAddress(address))}>
-                    <StaticAddress address={address} />
+                    <StaticAddress address={address} showProviderIcon />
```

Comment:
Does this mean we are always passing `showProviderIcon` as `true` for `<StaticAddress />` now?

Reply:
in current place - yes
need this flag because of we should show this icon not for all places when we reuse this code
and checking for show/hide PP icon inside [the component](https://github.com/bigcommerce/checkout-js/pull/1423/files#diff-ded4d841a24b30f3c3ba48ec964a53a61045c8156846829a1aab4bb7ee986ed9R49)

Reply:
Can we not just make the decision to show the label if the address is from paypal instead of adding an optional prop? Since we check if the address is from Paypal or not, I think that check should be good enough 🤔

Reply:
the same components `StaticAddress` with the same addresses but one with icon and another without
<img width="753" alt="Screenshot 2023-08-16 at 15 58 21" src="https://github.com/bigcommerce/checkout-js/assets/9430298/99f893fe-3976-4e7e-9813-f03eda28aedc">
<img width="752" alt="Screenshot 2023-08-16 at 15 58 41" src="https://github.com/bigcommerce/checkout-js/assets/9430298/6896328e-a2ea-4a02-ae1c-c9e28d383b2c">

Reply:
PayPal approved to add icon to each places with this component, so will remove this flag